### ISSUE TITLE
Default values on page load

### ISSUE THREAD

**Issue by zachalbert** [drewish/how-long#1]
When the page loads, the first sample value field should have focus by default. Also the sample time should be set to now (what do you think? That time field seems a tad finicky and starting with a value at least models how to use it...).

**Comment by drewish** [comment]
Agreed. I was thinking it might make sense to use the current time (auto updating) as the placeholder text. 

Occasionally you might want to stick a historical value in. But I think the majority of the use will be with the current time.
